web/kingpinflag/flag.go

```diff
@@ -28,13 +28,13 @@ func AddFlags(a *kingpin.Application, defaultAddress string) *web.FlagConfig {
 		systemdSocket = a.Flag(
 			"web.systemd-socket",
 			"Use systemd socket activation listeners instead of port listeners (Linux only).",
-		).Bool()
+		).HintOptions("true", "false").Bool()
```

Comment:
How does this work for kingpin Bool flags? They have a unique style of `--[no-]web.systemd-socket`. Not `--web.systemd-socket=true`.

Reply:
I removed it, it doesn't work for bool flags